server/controllers/finance/journal/index.js

```diff
@@ -13,10 +13,74 @@
 
 // module dependencies
 const db = require('../../../lib/db');
+const core = require('./core');
+const uuid = require('node-uuid');
+const journal = require('../journal/voucher');
 
 // expose to the api
 exports.list = list;
 
+exports.reverse = reverse;
+
+
+// GET /Grade
+function lookReverseTransaction(uid, user_id, creditNote) {
```

Comment:
Why is this called `lookReverseTransaction()` if it has an `INSERT` statement?  A better name would be `createReverseTransaction()` or something like that.